artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -3516,10 +3516,8 @@ public Queue createQueue(final QueueConfiguration queueConfiguration, boolean ig
       final AddressSettings addressSettings = addressSettingsRepository.getMatch(getRuntimeTempQueueNamespace(queueConfiguration.isTemporary()) + queueConfiguration.getAddress().toString());
       QueueConfigurationUtils.applyDynamicQueueDefaults(queueConfiguration, addressSettings);
 
-      if (ActiveMQServerLogger.LOGGER.isEnabled(Logger.Level.WARN)) {
-         if (new AddressImpl(queueConfiguration.getAddress(), configuration.getWildcardConfiguration()).containsWildCard() && addressSettings.getPageStoreName() == null) {
-            ActiveMQServerLogger.LOGGER.wildcardRoutingWithoutSharedPageStore(queueConfiguration.getName(), queueConfiguration.getAddress());
-         }
+      if (AddressImpl.isContainsWildCard(queueConfiguration.getAddress(), configuration.getWildcardConfiguration())) {
```

Comment:
I don't like import static, but for those who like it. .you could use an import static here instead.